docs/servicenow.md

```diff
@@ -0,0 +1,149 @@
+# ticketutil.servicenow
+
+This document contains information on the methods available when working
+with a ServiceNow object. A list of the ServiceNow fields that have been
+tested when creating and editing tickets will be included. Because each
+instance of ServiceNow can have custom fields and custom values, some
+of the tested fields may not be applicable to certain instances of ServiceNow.
+Additionally, your ServiceNow instance may contain ticket fields that we have
+not tested. Keep in your mind different ServiceNow tables contain different
+fields with different properties. Mandatory field of one table is optional
+in another table (or it doesn't exist at all).
+
+Custom field names and values can be passed in as keyword arguments when
+creating and editing tickets. These are processed by the ServiceNow REST API
+right afterwards. You can also use [get_ticket_content](#content) to get ticket
+content including preset field names.
+
+For more information about REST API and fields see ServiceNow's Wiki:
+http://wiki.servicenow.com/index.php?title=REST_API
```

Comment:
These three links appear on the same line when viewing the md file. Add line breaks to each one.